scheduler/scheduler-api/src/main/java/org/ow2/proactive/scheduler/common/job/Job.java

```diff
@@ -296,4 +316,23 @@ public String display() {
                "\tUserSpace = '" + userSpace + '\'' + nl + "\tVariables = {" + nl +
                Joiner.on('\n').withKeyValueSeparator("=").join(variables) + nl + "}";
     }
+
+    /**
+     * @return job deadline if it exists
+     * (currently deadline can be set as job GI vairable in ISO8601 date format)
+     */
+    public Optional<Date> getJobDeadline() {
+        if (genericInformation.containsKey(JOB_DDL)) {
+            try {
+                return Optional.of(ISODateTimeFormat.dateTimeNoMillis()
```

Comment:
My comment disappeared

Reply:
It is because Mykhailenko updated the code. Your comment is still available in the conversation tab by clicking on outdated : 
"In ExtendedSchedulingPolicy, we use the class ISO8601DateUtil for START_AT value, is the format used by joda ISODateTimeFormat the same ? Otherwise, it will create confusion."